src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/ImportOrganizationUserCommand.cs

```diff
@@ -16,6 +16,8 @@
 using Bit.Core.Repositories;
 using Bit.Core.Services;
 
+#nullable enable
```

Comment:
Should be the first line in the file.
The other new files here need this as well, even models.